src/main/java/com/databasepreservation/modules/siard/out/metadata/MetadataExportStrategy.java

```diff
@@ -0,0 +1,13 @@
+package com.databasepreservation.modules.siard.out.metadata;
+
+import com.databasepreservation.model.exception.ModuleException;
+import com.databasepreservation.model.structure.DatabaseStructure;
+import com.databasepreservation.modules.siard.common.SIARDArchiveContainer;
+
+/**
+ * @author Bruno Ferreira <bferreira@keep.pt>
+ */
+public interface MetadataExportStrategy {
+	void writeMetadataXML(DatabaseStructure databaseStructure, SIARDArchiveContainer container) throws ModuleException;
+	void writeMetadataXSD(DatabaseStructure databaseStructure, SIARDArchiveContainer container) throws ModuleException;
```

Comment:
Add write strategy as an argument to both metadata export strategy methods. Do the same of the metadata import strategy.